staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -128,100 +129,108 @@ func (c *KubeSchedulerConfiguration) EncodeNestedObjects(e runtime.Encoder) erro
+	// filtering succeeds or not.
 	PostFilter *PluginSet `json:"postFilter,omitempty"`
 
-	// PreScore is a list of plugins that are invoked before scoring.
+	// A list of Plugins that are invoked before the "Score" phase.
 	PreScore *PluginSet `json:"preScore,omitempty"`
 
-	// Score is a list of plugins that should be invoked when ranking nodes that have passed the filtering phase.
+	// A list of Plugins that should be invoked when ranking nodes that
+	// have passed the "Filter" phase.
 	Score *PluginSet `json:"score,omitempty"`
 
-	// Reserve is a list of plugins invoked when reserving/unreserving resources
-	// after a node is assigned to run the pod.
+	// A list of Plugins invoked when reserving/unreserving resources after a node
+	// is assigned to run a Pod.
 	Reserve *PluginSet `json:"reserve,omitempty"`
 
-	// Permit is a list of plugins that control binding of a Pod. These plugins can prevent or delay binding of a Pod.
+	// A list of Plugins that control the binding of a Pod. These Plugins can
+	// prevent or delay binding of a Pod.
 	Permit *PluginSet `json:"permit,omitempty"`
 
-	// PreBind is a list of plugins that should be invoked before a pod is bound.
+	// A list of Plugins that should be invoked before a Pod is bound.
 	PreBind *PluginSet `json:"preBind,omitempty"`
 
-	// Bind is a list of plugins that should be invoked at "Bind" extension point of the scheduling framework.
-	// The scheduler call these plugins in order. Scheduler skips the rest of these plugins as soon as one returns success.
+	// A list of Plugins that should be invoked at "Bind" extension point of
+	// the scheduling framework.
+	// The scheduler executes these Plugins in order until the first Plugin that
+	// returns success.
 	Bind *PluginSet `json:"bind,omitempty"`
 
-	// PostBind is a list of plugins that should be invoked after a pod is successfully bound.
+	// A list of Plugins that should be invoked after a Pod is successfully bound.
 	PostBind *PluginSet `json:"postBind,omitempty"`
 }
 
-// PluginSet specifies enabled and disabled plugins for an extension point.
+// PluginSet specifies enabled and disabled Plugins for an extension point.
 // If an array is empty, missing, or nil, default plugins at that extension point will be used.
 type PluginSet struct {
-	// Enabled specifies plugins that should be enabled in addition to default plugins.
-	// These are called after default plugins and in the same order specified here.
+	// The Plugins that should be enabled in addition to default Plugins.
+	// These are called after default Plugins and in the same order specified here.
 	// +listType=atomic
 	Enabled []Plugin `json:"enabled,omitempty"`
-	// Disabled specifies default plugins that should be disabled.
-	// When all default plugins need to be disabled, an array containing only one "*" should be provided.
+	// The default Plugins that should be disabled.
+	// In particular, a single `*` means to disable all Plugins.
```

Comment:
a single plugin with the name field as '*' right? I know the previous doc didn't clarify this either

Reply:
Fixed.